source/collections/SortedVectorList.ooc

```diff
@@ -0,0 +1,29 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+use collections
+
+SortedVectorList: class extends VectorList<Int> {
```

Comment:
Gaaaaaaaaaaaaaaaaaaaaaaaaaaaah I had all sorts of weird problems until I discovered `extends VectorList<Int>`